geopandas/geodataframe.py

```diff
@@ -1627,7 +1628,11 @@ def dissolve(
             if a non-default value is given for this parameter.
 
             .. versionadded:: 0.9.0
+        **kwargs: Keyword arguments to be passed as arguments to aggfunc.
+            In particular, `numeric_only` may be supplied, which will
+            may be required for pandas 2.0 for certain aggfuncs.
```

Comment:
I think we can be more explicit about that it is passed to the `aggregate()` method on the groupby object?